test/webpack.tests.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+
+const testsContext = require.context('.', true, /\-test\.js$/);
```

Comment:
I'm not convinced that we need Karma messing with the test build at all. What's the advantage of doing that rather than using the same build tools and process we use for the rest of the repository assets?

I'm completely failing to see the logic of using Karma for the test build, except that karma also wants to bundle watch functionality that is probably better done outside the test config, too.. see my other comments about using npm scripts instead of Karma to automate all of that in a way that is consistent across the entire project, not just the tests.

Two different ways to configure the same things just seems like adding complexity with zero net gain.